docker-compose.yaml

```
@@ -18,3 +18,43 @@ services:
       - POSTGRES_DB=${COMPOSE_PROJECT_NAME}-dev
       - POSTGRES_USER=${COMPOSE_PROJECT_NAME}-user
       - POSTGRES_PASSWORD=manager
+
+  # ###########################################################################
+  # Localstack - solution for many aws services in one container
+  # Used for s3
+  # Docs: https://docs.localstack.cloud/overview/
+  # ###########################################################################
+  localstack-services:
+    image: localstack/localstack:3.3.0
+    hostname: localhost.localstack.cloud
+    ports:
+      - "4566:4566"
+      - '9000:9000'
+      - '9001:9001'
```

```suggestion
      - "9000:9000"
      - "9001:9001"
```
By the way, what is it for? I thought that localstack doesn't provide UI to work with